pkg/monitor/cluster/clusterversions.go

```diff
@@ -38,14 +39,21 @@ func (mon *Monitor) emitClusterVersions(ctx context.Context) error {
 		availableRP = version.GitCommit
 	}
 
+	actualVersion := actualVersion(cv)
+	actualMinorVersion := ""
+	if len(actualVersion) > 0 {
+		actualMinorVersion = actualVersion[0:strings.LastIndex(actualVersion, ".")]
+	}
```

Comment:
I think version can be `4.3.0-0.nightly-2020-04-17-062811` where last `.` is part of suffix.

You might want to use `version.ParseVersion(...)` instead.